### PR TITLE
Fixed input

### PR DESCRIPTION
**Pull request by anshtripathi01**
input preview

### CODE REVIEW

File: Components/Form/form.css [hunk 1]
```diff
@@ -11,7 +11,12 @@
     padding: 10px;
     margin: 10px;
 }
-
+.error-msg{
+    color: #ff0000;
```
Comment (AtulHande03a) [inline Components/Form/form.css:15]: add variable name to the color.

**Review by AtulHande03a — APPROVED** [review]
rest looks good !!

File: Components/Form/form.css (hunk 1 above)
Comment (anshtripathi01) [reply Components/Form/form.css:15]: added

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to merge

**Comment by anshtripathi01** [comment]
Thank you for review

**Pull request merged**